packages/web/components/ui/media-embed-node.tsx

```
@@ -64,7 +63,7 @@ export const MediaEmbedElement = withHOC(
                 isYoutube ? (
                   <LiteYouTubeEmbed
                     id={embed!.id!}
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)